packetbeat/packetbeat.reference.yml

```diff
@@ -433,6 +433,10 @@ packetbeat.protocols:
   # certificate chains are sent to Elasticsearch. The default is true.
   #send_certificates: true
 
+  # If this option is enabled, the raw certificates will be stored
+  # in PEM format under the `raw` key. The default is false.
+  #include_raw_certificates: false
```

Comment:
This one is generated, you need to modify `packetbeat/_meta/beat.reference.yml` instead and run `make update`. That's why you get a check failure.